src/styles/main.scss

```diff
@@ -1,3 +1,14 @@
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
+@import "./blocks/calendar.scss";
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
Will be better to move it to some class, instead of setting all styles directly to `body` tag